runtime/runtime-params-estimator/estimator-warehouse/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "estimator-warehouse"
+version = "0.0.0"
+authors = ["Near Inc <hello@nearprotocol.com>"]
+# Please update rust-toolchain.toml as well when changing version here:
+edition = "2021"
+publish = false
+rust-version = "1.56.0"
+
+[dependencies]
+anyhow = "1"
+chrono = "0.4"
+clap = {version = "3.1", features = ["derive"]}
+reqwest = {version = "0.11", features = ["blocking", "json"]}
+rusqlite = {version = "0.27", features = ["bundled", "chrono"]}
+serde = {version = "1", features = ["derive"]}
+serde_json = "1"
```

Comment:
Minor, but, based on the following discussion:

https://users.rust-lang.org/t/psa-please-specify-precise-dependency-versions-in-cargo-toml/71277

I now prefer to always specify all three numbers in Cargo.toml